packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -957,8 +938,7 @@ export function performConcurrentWorkOnRoot(
               originallyAttemptedLanes,
               errorRetryLanes,
             );
-            // We assume the tree is now consistent because we didn't yield to any
-            // concurrent events.
+            renderWasConcurrent = false;
```

Comment:
This is just for clarity, right? As far as I can tell, it's not read anymore at this point.

Reply:
yeah :D in case a `continue` gets added later in the block or something